binaries/cuprated/src/main.rs

```diff
@@ -13,10 +13,16 @@
 
 mod blockchain;
 mod config;
+mod constants;
 mod p2p;
 mod rpc;
+mod statics;
 mod txpool;
 
 fn main() {
+    // Initialize global static `LazyLock` data.
+    statics::init_lazylock_statics();
```

Comment:
The statics get initialized here.

We could add this to a broader `init_everything()` later that setups up logging as too.

Reply:
for now this will be ok